contracts/pools/PoolCollection.sol

```diff
@@ -350,19 +342,15 @@ contract PoolCollection is IPoolCollection, Owned, BlockNumber, Utils {
             tradingFeePPM: _defaultTradingFeePPM,
             tradingEnabled: false,
             depositingEnabled: true,
-            averageRate: AverageRate({ blockNumber: 0, rate: zeroFraction112() }),
-            depositLimit: 0,
+            averageRates: AverageRates({ blockNumber: 0, rate: zeroFraction112(), invRate: zeroFraction112() }),
```

Comment:
At present, function `depositFor` calls function `_updateTradingLiquidity` with `rate`.
A possible issue is when `rate` and `invRate` are far apart from each other (i.e., not close to being inverses of each other).
Depending on which one of them is used, the outcome may be very different.
The impact of this is mostly on the output trading liquidities (of BNT and of TKN).

Reply:
It's not different than before. The protocol allows depositing liquidity at any rate (even when there is no rate during the bootstrapping) phase. Still, it won't be possible to withdraw as long as the pool is unstable. Noted.